opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/view.py

```diff
@@ -133,6 +158,9 @@ def __init__(
         self._description = description
         self._attribute_keys = attribute_keys
         self._aggregation = aggregation or self._default_aggregation
+        self._exemplar_reservoir_factory = (
+            exemplar_reservoir_factory or default_reservoir_factory
```

Comment:
Little bit confused at to what we want here stored. Should `self._exemplar_reservoir_factory` be a  `ExemplarReservoirFactory` or a callable that returns a `ExemplarReservoirFactory`? Does this match what is being expected in `_Aggregation` __init__?